packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/actions/shuffleThis.yml

```diff
@@ -12,16 +12,12 @@ initialState:
     - anchor: {line: 0, character: 0}
       active: {line: 0, character: 5}
   marks: {}
+fullTargets: [{type: primitive, mark: {type: cursor}, selectionType: token, position: contents, insideOutsideType: inside, modifier: {type: identity}, isImplicit: false}]
 finalState:
   documentContents: hello
   selections:
     - anchor: {line: 0, character: 0}
       active: {line: 0, character: 5}
-  thatMark:
-    - type: UntypedTarget
-      contentRange:
-        start: {line: 0, character: 0}
-        end: {line: 0, character: 5}
-      isReversed: false
-      hasExplicitRange: true
```

Comment:
Not that we care for this test, but I wonder why it dropped the `that` mark. Did you just run update fixtures?

Reply:
Correct

Reply:
But in two stages. Didnt revert in between.

Reply:
Reverting back to the original and running update fixtures once kept the that mark.

Reply:
Huh interesting. Yeah I guess that makes sense. Once it became an error the fact that it should preserve that mark was lost. Not sure how to solve that one but prob not a big deal